sources/proc_common.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	numRegexPattern = regexp.MustCompile(`[0-9]*\.[0-9]+|[0-9]+`)
```

Comment:
IIRC, the double backslash was converted to a single backslash based on some previous conversation somewhere in the repo, yes?